frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-patient-count/radiogram-card-content-patient-count.component.ts

```diff
@@ -0,0 +1,53 @@
+import type { OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { createSelector, Store } from '@ngrx/store';
+import { PatientCountRadiogram } from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import type { AppState } from 'src/app/state/app.state';
+import {
+    createSelectSimulatedRegion,
+    selectConfiguration,
+} from 'src/app/state/application/selectors/exercise.selectors';
+
+@Component({
+    selector: 'app-radiogram-card-content-patient-count',
+    templateUrl: './radiogram-card-content-patient-count.component.html',
+    styleUrls: ['./radiogram-card-content-patient-count.component.scss'],
+})
+export class RadiogramCardContentPatientCountComponent implements OnInit {
+    @Input() radiogram!: PatientCountRadiogram;
+    bluePatientsEnabled$!: Observable<boolean>;
+    simulatedRegionName$!: Observable<string>;
+    totalPatientCount!: number;
+
+    constructor(private readonly store: Store<AppState>) {}
+
+    ngOnInit(): void {
+        this.totalPatientCount =
+            this.radiogram.patientCount.black +
+            this.radiogram.patientCount.white +
+            this.radiogram.patientCount.red +
+            this.radiogram.patientCount.yellow +
+            this.radiogram.patientCount.green +
+            this.radiogram.patientCount.blue;
+
+        const bluePatientsEnabledSelector = createSelector(
+            selectConfiguration,
+            (configuration) => configuration.bluePatientsEnabled
+        );
+        this.bluePatientsEnabled$ = this.store.select(
+            bluePatientsEnabledSelector
+        );
```

Comment:
I don't think you need to create a separate variable for this, right?

Reply:
yes but it is more readable that way imo

Reply:
I do not see where the existing version is any worse than the suggested change, so I would like to leave it as is

Reply:
We've done it this way everywhere else in this project.

Reply:
This is not true look at `radiogram-card.component.ts` Line 83-95 or Line 57-58 or simulated-region-overview-tab.component.ts Lines 79 -85 So I think this code style choice is one that is up to the dev.